packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -237,6 +244,17 @@ export default class CFDStore extends BaseStore {
                 } else this.setJurisdictionSelectedShortcode('svg');
                 this.demoCFDSignup();
             }
+        } else if (platform === CFD_PLATFORMS.DERIVEZ) {
+            this.setJurisdictionSelectedShortcode('svg');
+            const values = {
+                platform,
+                account_type: this.account_type.category,
+                market_type: this.account_type.type,
+                company: this.jurisdiction_selected_shortcode,
+            };
+            this.openCFDAccount(values);
+            this.enableCFDPasswordModal();
```

Comment:
put the condition for when account creation fails as well.
const response = await this.openMT5Account(values);
if (!response.error) { your code } else { this.setError(true, response.error) }